Config/TestNG.xml

```diff
@@ -1,16 +1,23 @@
  <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
 <suite name="All Test Cases" verbose="1" >
-    <test name="Verify user is logged in">
-        <parameter name="testRunId1" value="TCTC-1"/>
+<!--    <test name="Verify user is logged in">-->
+<!--        <parameter name="testRunId1" value="TCTC-1"/>-->
+<!--        <classes>-->
```

Comment:
Why Commented code is pushed?

Reply:
I was unaware about not pushing the commented code, Did comment this part of code in order to run only LeavePageTest

Reply:
@SachinKumar1606 I was unaware about not pushing the commented code, Did comment this part of code in order to run only LeavePageTest